ddl/ddl_api.go

```diff
@@ -1133,16 +1133,32 @@ func getPrimaryKey(tblInfo *model.TableInfo) *model.IndexInfo {
 			// table has explicit primary key
 			return key
 		}
+		// The case index without any columns should never happen, but still do a check here
+		if key.Columns == nil || len(key.Columns) == 0 {
```

Comment:
```suggestion
		if len(key.Columns) == 0 {
```

only one condition will be ok?